### PR TITLE
Judicial system/Now using strict TS

### PR DESCRIPTION
**Pull request by oddsson**
# Judicial system's web project now uses strict TS.

https://app.asana.com/0/322488613152836/1199143603683037

## What

Enable strict TS setting in Judicial-system-web

## Why

Better code quality. 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Comment by gudjong** [comment]
I tested the web locally and everything seems to work well.

File: apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx [hunk 1]
```diff
@@ -226,10 +226,10 @@ export const JudgeOverview: React.FC = () => {
                   , verði með úrskurði Héraðsdóms Reykjavíkur gert að sæta
                   gæsluvarðhaldi til
                   <Text as="span" fontWeight="semiBold">
-                    {` ${formatDate(
-                      workingCase.requestedCourtDate,
-                      'EEEE',
-                    ).replace('dagur', 'dagsins')} 
+                    {` ${
+                      formatDate(workingCase.requestedCourtDate, 'EEEE') ||
+                      ''.replace('dagur', 'dagsins')
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx:231]: Cool, I fixed this potential null pointer with a question mark 'formatDate(...)?.replace(...)'. Both approaches work.

Comment (gudjong) [reply apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx:231]: We could also let formatDate return '' instead of undefined.

**Review by gudjong — APPROVED** [review]
Very nice.

**Review by addi — APPROVED** [review]

File: apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx [hunk 2]
```diff
@@ -261,10 +261,17 @@ export const JudgeOverview: React.FC = () => {
                   , verði með úrskurði Héraðsdóms Reykjavíkur gert að sæta
                   gæsluvarðhaldi til
                   <Text as="span" fontWeight="semiBold">
+<<<<<<< HEAD
```
Comment (addi) [inline apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx:264]: !!!

**Review by addi — APPROVED** [review]

**Review by gudjong — APPROVED** [review]

**Pull request merged**